frontend/tests/search.spec.ts

```diff
@@ -110,5 +110,40 @@ test("Read access is checked", async ({ page, standardData, activeSearchIndex, b
     });
 });
 
+for (const username of ["jose", "admin"] as const) {
+    test(`Video search finds listed & writable (for ${username})`, async ({
```

Comment:
I wonder if this should be in a separate file. I know I haven't set the best example for this, but I think it is generally encouraged to split these tests up into smaller packages and "domains", so to speak. Right now this mixes the general search (for the search page) with the `searchableSelect` things used for adding/editing content.

I would probably rename this file to `searchPage.spec` and do the unrelated tests either in `blocks.spec` or rather in a new file like `searchableSelect.spec` **in the long run**. So no need to change it in this PR (and maybe you disagree - this is something we can discuss at a later point).

Reply:
Yeah that's a good point. Probably move it to its own file. Also because it can run different files in parallel.